### ISSUE TITLE
Accordion bug within addEventListener target

### ISSUE THREAD

**Issue by be-codified** [w3c/aria-practices#425]
Just noticed, when using Chrome and clicking on panel's title, target is:

`<span class="Accordion-title">Personal Information</span>`

instead of: 

```
<button aria-expanded="false" class="Accordion-trigger" aria-controls="sect1" id="accordion1id">
    <span class="Accordion-title">Personal Information</span><span class="Accordion-icon"></span>
 </button>
```
This will fix it:

```
accordion.addEventListener('click', (event) => {
    const target = event.target.classList.contains('Accordion-trigger') ? event.target : 
    event.target.parentNode;
    ...
```

Well played in Firefox and Safari though.

**Comment by gerardkcohen** [comment]
@be-codified I have a fix coming soon.

**Comment by mcking65** [comment]
Fixed with commit 74887d9. Many thanks to @gerardkcohen!
